plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraLatestConnectorSmokeTest.java

```diff
@@ -38,17 +41,48 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
             case SUPPORTS_CREATE_SCHEMA:
                 return false;
 
+            case SUPPORTS_CREATE_VIEW:
+                return false;
+
             case SUPPORTS_RENAME_TABLE:
+            case SUPPORTS_RENAME_TABLE_ACROSS_SCHEMAS:
+                return false;
+
+            case SUPPORTS_ARRAY:
                 return false;
 
-            case SUPPORTS_DELETE:
-                return true;
+            case SUPPORTS_COMMENT_ON_TABLE:
+            case SUPPORTS_COMMENT_ON_COLUMN:
+                return false;
 
-            case SUPPORTS_ROW_LEVEL_DELETE:
+            case SUPPORTS_TOPN_PUSHDOWN:
                 return false;
 
             default:
                 return super.hasBehavior(connectorBehavior);
         }
     }
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        CassandraServer server = closeAfterClass(new CassandraServer("3.11.10"));
+        CassandraSession session = server.getSession();
+        createTestTables(session, KEYSPACE, Timestamp.from(TIMESTAMP_VALUE.toInstant()));
+        return createCassandraQueryRunner(server, ImmutableMap.of(), REQUIRED_TPCH_TABLES);
```

Comment:
nit: You don't need to move this method for minimizing diff.

Reply:
Thank you.  I have corrected this inconsistency.